_i18n/fr/get-started/faq.md

```diff
@@ -60,7 +60,7 @@ Si vous faites fonctionner un nœud complet localement, vous devez copier la tot
 
 <div class="tab-content" markdown="1">
 
-Pour un portefeuille léger, vous communiquer votre clef publique d'audit à un nœud, qui recherche pour vous les transactions qui vous sont adressées sur la chaîne de blocs. Ce nœud saura quand vous aurez reçu des fonds, mais ne saura pas combien, de la part de qui, ou à destination de qui. Selon l'application de portefeuille, vous pourriez pouvoir disposer d'un nœud que vous contrôleriez pour vous prémunir des fuites de confidentialité. Pour plus de confidentialité, utilisez un portefeuille traditionnel qui peut être utilisé avec votre propre nœud.
+Pour un portefeuille léger, vous communiquez votre clef publique d'audit à un nœud qui recherche pour vous les transactions qui vous sont adressées sur la chaîne de blocs. Ce nœud saura quand vous aurez reçu des fonds, mais ne saura pas combien vous avez reçu, de la part de qui vous les avez reçu, ou à destination de qui vous aurez envoyé des fonds. Selon l'application de portefeuille, vous pourriez pouvoir disposer d'un nœud que vous contrôleriez pour vous prémunir des fuites de confidentialité. Pour plus de confidentialité, utilisez un portefeuille traditionnel qui peut être utilisé avec votre propre nœud.
```

Comment:
vous les avez reçu -> reçus

This sentence is quite unclear: "Selon l'application de portefeuille, vous pourriez pouvoir disposer d'un nœud que vous contrôleriez pour vous prémunir des fuites de confidentialité"

Reply:
ok reçus.

"Depending on your wallet software, you may be able to use a node you control to avoid privacy leaks."
So we could improve perhaps to:
"Selon l'application de portefeuille utilisée, vous pourriez disposer d'un nœud sous votre contrôle vous prémunissant des fuites de confidentialité"
What do you think?

Reply:
I suggest: Certains portefeuilles légers permettent d'utiliser un nœud que vous contrôlez et d'éviter ainsi des fuites de confidentialité (or maybe: de données personnelles)?